src/parseline.c

```diff
@@ -846,23 +846,29 @@ void m_nick (char *sender, char *tail)
 
     uptr2 = find_user(newnick);
     if (uptr2) {
-        if (uptr && (find_link2(oldnick,newnick) || find_link2(newnick,oldnick)) && uptr->authed == 1) {
+        if (uptr && (find_link2(oldnick, newnick) || find_link2(newnick, oldnick)) && uptr->authed == 1)
+        {
```

Comment:
Nit: coding style (no newline before opening bracket in branches).

Reply:
Fixed